src/args.rs

```diff
@@ -28,14 +28,14 @@ use structopt;
 #[derive(StructOpt, Debug, Clone)]
 #[structopt(
     template = "USAGE: [FLAGS] [OPTIONS] [FILTER]\n\n{all-args}\n\n\n{after-help}",
-    raw(setting = "structopt::clap::AppSettings::DisableVersion"),
+    setting = structopt::clap::AppSettings::DisableVersion,
     after_help = "By default, all tests are run in parallel. This can be altered with the \n\
         --test-threads flag or the RUST_TEST_THREADS environment variable when running \n\
         tests (set it to 1).\n\
         \n\
         All tests have their standard output and standard error captured by default. \n\
         This can be overridden with the --nocapture flag or setting RUST_TEST_NOCAPTURE \n\
-        environment variable to a value other than \"0\". Logging is not captured by default.",
+        environment variable to a value other than \"0\". Logging is not captured by default."
```

Comment:
Can you re-add all the trailing comma's? Not sure if you removed them on purpose or if rustfmt did that automatically? But AFAIK, trailing commas in multi-line lists are the way to go. There are multiple occurrences below.

Reply:
Sure, I guess `rustfmt` was slightly too aggressive.